src/components/Navbar/index.jsx

```diff
@@ -33,27 +37,64 @@ const socialIcons = [
 ]
 
 export default function Navbar() {
+  const { toggleTheme, theme } = useTheme()
```

Comment:
**Minor:** For clarity. Always have the contexts/hooks close to each other. Maybe we can add that to prettier

```
useState()
useTheme()
useAuth()

variables

methods

return  (
 <div>
 </div
 )
```